modules/engine/src/test/scala/observe/engine/SequenceSuite.scala

```diff
@@ -208,7 +204,7 @@ class SequenceSpec extends AnyFlatSpec {
   val seqzar0: Sequence.Zipper[IO]                                      = simpleSequenceZipper(stepzar0)
   val seqzar1: Sequence.Zipper[IO]                                      = simpleSequenceZipper(stepzar1)
 
-  "next" should "be None when there are no more pending executions" in {
+  test("be None when there are no more pending executions") {
```

Comment:
In this test and the following, we lose a bit of information in the test description by omitting the name of the tested function.